test/expected/append-10.out

```diff
@@ -1044,6 +1044,30 @@ ORDER BY time DESC;
                      Heap Fetches: 129
 (31 rows)
 
+-- test runtime exclusion does not activate for constraints on non-partitioning columns
+-- should not use runtime exclusion
+:PREFIX SELECT * FROM append_test a1 LEFT JOIN LATERAL(SELECT * FROM append_test a2 WHERE a1.colorid = a2.colorid ORDER BY time DESC LIMIT 1) a2 ON true ORDER BY a1.time;
+                                                          QUERY PLAN                                                          
+------------------------------------------------------------------------------------------------------------------------------
+ Nested Loop Left Join (actual rows=5 loops=1)
+   ->  Custom Scan (ChunkAppend) on append_test a1 (actual rows=5 loops=1)
+         Order: a1."time"
+         ->  Index Scan Backward using _hyper_1_1_chunk_append_test_time_idx on _hyper_1_1_chunk a1_1 (actual rows=2 loops=1)
+         ->  Index Scan Backward using _hyper_1_2_chunk_append_test_time_idx on _hyper_1_2_chunk a1_2 (actual rows=2 loops=1)
+         ->  Index Scan Backward using _hyper_1_3_chunk_append_test_time_idx on _hyper_1_3_chunk a1_3 (actual rows=1 loops=1)
+   ->  Limit (actual rows=1 loops=5)
+         ->  Custom Scan (ChunkAppend) on append_test a2 (actual rows=1 loops=5)
```

Comment:
How does this show that runtime exclusion is off as opposed to that it just didn't exclude anything

Reply:
It would have a line like this: `Chunks excluded during runtime: 4`